cookbook/frontend/bower.rst

```diff
@@ -0,0 +1,125 @@
+.. index::
+    single: Front-end; Bower
+
+Using Bower with Symfony
+========================
+
+Symfony and all its packages are perfectly managed by Composer. Bower is a
+dependency management tool for front-end dependencies, like Bootstrap or
+jQuery. As Symfony is purely a back-end framework, it can't help you much with
+Bower. Fortunately, it is very easy to use!
+
+Installing Bower
+----------------
+
+Bower_ is built on top of NodeJS_. Make sure you have that installed and
+then run:
+
+.. code-block:: bash
+
+    $ npm install -g bower
+
+After this command succeeded, run ``bower`` in your terminal to find out if
+it's installed correctly.
+
+.. tip::
+
+    If you don't want to have NodeJS on your computer, you can also use
+    BowerPHP_ (an unofficial PHP port of Bower). Beware that this is still in
+    an alpha state. If you're using BowerPHP, use ``bowerphp`` instead of
+    ``bower`` in the examples.
+
+Configuring Bower in your Project
+---------------------------------
+
+By default, Bower will install all front-end dependencies in a
+``bower_packages/`` directory in the root of your project. In Symfony, only the
+``web/`` directory has public access, so you have to configure Bower to install
+it under the ``web/`` directory instead.
+
+A common practice is to put your front-end dependencies in
+``web/assets/vendor/``. You can configure this directory in a ``.bowerrc`` file
+in the root of your project. It's using the JSON syntax:
```

Comment:
I want to keep this article as short as possible, just to show the Symfony-specific stuff for Bower and avoid duplication (but still be as clear as possible). You're already actually doing a pretty good job with this. But I think we can shorten this section, something like:

```rst
Normally, Bower downloads everything into a ``bower_components/`` directory. In
Symfony, only files in the ``web/`` directory are publicly accessible, so you need
to configure Bower to download things there instead. To do that, just create a ``.bowerrc``
file with a new destination (like ``web/assets/vendor``):
```